docs/src/main/sphinx/release/release-354.md

```diff
@@ -0,0 +1,80 @@
+# Release 354 (XXXXXXXX Mar 2021)
+
+## General
+
+* Preferred write partitioning is now triggered automatically when the number
+  of written partitions exceeds or is equal to
+  ``preferred-write-partitioning-min-number-of-partitions``. The number of written
+  partitions is estimated from stats, therefore CTAS or insert source tables needs to have
+  statistics collected. To forcefully enable preferred partitioning,
+  ``preferred-write-partitioning-min-number-of-partitions`` needs to be set to
+  ``0`` or ``1``. To fully disable preferred write partitioning, set
```

Comment:
> ``0`` or ``1``

can we simplify and set only one value?

@sopel39 let's have `1` here and let's add `@Min("1")` to that property to prevent confusion.
it's not good when two different values mean the same.

Reply:
https://github.com/trinodb/trino/pull/7348